### ISSUE TITLE
If someone closes their window mid-game, the game soft-locks - when it gets to their turn, nothing can happen.

### ISSUE THREAD

**Issue by id-hkj** [id-hkj/Sequence-Multiplayer#8]
There should also be the addition of a back button so you are able to return to the previous screen (especially if the game has ended).

**Comment by id-hkj** [comment]
Good evening id-hkj,
Here at Little Room Projects, we have decided that this feature will not be added. This is because we are a small team of one person (we also have consultants - friends who we show our projects to in order to check they look good/are easy to use) with very limited resources. We ran into a bit of a roadblock when figuring out how to make this work. The issue was to do with the server, and our lead programmer is not that familiar with that field. Due to this, we have decided not to work on the project further as it would be a waste of time and effort - the application is still very usable.
There are other projects we are keen to start work on, and now that we are happy to call this one finished (at least for now), we look forward to sharing them with you in the future.
Yours Sincerely,
id-hkj (CEO of Little Room Projects)
